rholang/examples/hello_world_again.rho

```diff
@@ -1,9 +1,9 @@
 // A simple HelloWorld contract demonstrating a contract being rerun
 
-new helloworld in {
+new helloworld, stdout(`rho:io:stdout`) in {
```

Comment:
Please also update rholangtut.md

Also please consider all text occurrences of @"stdout", @"stdoutAck", etc. - I think there's some default testing rholang hardcoded in the p2p test tool, for example...